types/office-js-preview/index.d.ts

```diff
@@ -11760,11 +11760,11 @@ declare namespace Office {
     /**
      * Represents the categories on an item.
      * 
-     * In Outlook, a user can group messages and appointments by using a category to color-code them.
+     * In Outlook, a user can tag messages and appointments by using a category to color-code them.
      * The user defines {@link Office.MasterCategories | categories in a master list} on their mailbox.
      * They can then apply one or more categories to an item.
      *
-     * **Important**: In Outlook on the web, you can't use the API to manage categories on a message in Read mode.
+     * **Important**: In Outlook on the web, you can't use the API to manage categories on a message in Compose mode.
```

Comment:
i would me more clear here. Manage in my world has more to do with categories CRUD (adding changing deleting updating.  lets me more assertive. (i.e. In OWA you cannot use the APIs to assign or remove categories to a message or appointment when in Componse mode). You can still add/remove categories to the mailbox master list.

Reply:
@JuaneloJuanelo To clarify, this note is on the Categories object and the OWA restriction is only on message Compose - the other modes should be able to manage categories on the item. Manage seems to be the correct term here since it is about categories CRUD on the item.